Header header
int8 mode
geometry_msgs/Vector3 pose_camera
geometry_msgs/Vector3 pose_base
geometry_msgs/Vector3 pose_odom
geometry_msgs/Vector3 pose_image
geometry_msgs/Vector3 angle
int16 armor_count
Armor[] armor_list